src/utils/separator.js

```diff
@@ -0,0 +1,19 @@
+import { CUSTOM_SEPARATOR } from "../constants/index.js";
+
+export const extractCustomSeparators = (customString) => {
+  // 커스텀 문자열이 없다면 빈 배열 early return
+  if (customString === "") {
+    return [];
+  }
+
+  const matches = customString.matchAll(CUSTOM_SEPARATOR);
+  return Array.from(matches, (match) => match[1]);
+};
+
+export const mergeSeparators = (...separators) => {
+  return [...separators];
+};
```

Comment:
separator 뿐 아니라 어떤 원소들이와도 합쳐줄수있으니 mergeSeparators보단 merge(...items)가 좀더 네이밍적으로 좋을것 같아요!
받은 원소들을 굳이 스프레드를 다시 시켜서 배열로 만드는 이유가 있을까요? 
```suggestion
export const mergeSeparators = (...separators) => {
  return separators;
};
```
이렇게도 가능하지 않나 싶어서요!

Reply:
오호 그 방식은 생각 못했습니다... 간결해서 좋은 것 같아요!